test/e2e/networks/ci.toml

```diff
@@ -1,10 +1,10 @@
 # This testnet is run by CI, and attempts to cover a broad range of
 # functionality with a single network.
 
-initial_height = 1000
+disable_legacy_p2p = false
```

Comment:
You can ignore most of these changes. My vsc TOML extension automatically sorts :)

Reply:
maybe we should all be running that extension, tbh

Reply:
yah I think we should ^__^